### ISSUE TITLE
[Conformer model] Training not converging 

### ISSUE THREAD

**Issue by arunbaby0** [espnet/espnet#3969]
When training a conformer model with [this config](https://github.com/espnet/espnet/blob/master/egs2/librispeech/asr1/conf/tuning/train_asr_conformer7_n_fft512_hop_length256.yaml) the losses don't seem to reduce and the WER is also not as expected. I have trained till 45 epoch. PFB the loss and WER images.
![wer](https://user-images.githubusercontent.com/16813485/149762586-57aeecb4-d5e5-4af0-81b2-9ee2d6bcf752.png)
![loss](https://user-images.githubusercontent.com/16813485/149762596-439c287c-3ce0-4d19-a36e-be2deeba91d4.png)

However for the exact same data transformer model [config](https://github.com/espnet/espnet/blob/master/egs2/librispeech/asr1/conf/tuning/train_asr_transformer3.yaml) was working perfectly fine.
![cer_ctc](https://user-images.githubusercontent.com/16813485/149762634-e3394d31-dfb6-4ce7-9102-06cd38e76ded.png)

Is there any config changes required while training?

**Comment by sw005320** [comment]
Which data did you use?

Conformer is more difficult to tune the hyperparameters than transformer.
This often happens.
I recommend you find a similar database in our recipe https://github.com/espnet/espnet/tree/master/egs2 and refer the conformer config based on it.

Also, we usually tune
- learning rate: https://github.com/espnet/espnet/blob/master/egs2/librispeech/asr1/conf/tuning/train_asr_conformer7_n_fft512_hop_length256.yaml#L49
- kernel size: https://github.com/espnet/espnet/blob/master/egs2/librispeech/asr1/conf/tuning/train_asr_conformer7_n_fft512_hop_length256.yaml#L30
- batch size (https://github.com/espnet/espnet/blob/master/egs2/librispeech/asr1/conf/tuning/train_asr_conformer7_n_fft512_hop_length256.yaml#L3 or https://github.com/espnet/espnet/blob/master/egs2/librispeech/asr1/conf/tuning/train_asr_conformer7_n_fft512_hop_length256.yaml#L4)

**Comment by arunbaby0** [comment]
Thanks, @sw005320 for the recommendations.

This is a private dataset of around 500hrs of data consisting of both read speech and conversational speech.

**Comment by arunbaby0** [comment]
Thanks, @sw005320, the learning rate tuning helped in converging.
